apps/system/js/devtools/logshake.js

```diff
@@ -196,12 +202,16 @@
           return navigator.mozL10n.get('logsOperationFailed',
                                        { operation: error.operation });
         }
+
+        if ('name' in error && error.name === 'NotFoundError') {
+          return navigator.mozL10n.get('logsNotFoundError');
+        }
       }
 
       return '';
     },
 
-    showErrorMessage: function(error, notif) {
+    showErrorMessage: function(title, error, notif) {
```

Comment:
I don't think it's big deal for now. Might make the code more complex for an edge case.